sirepo/server.py

```diff
@@ -697,6 +689,18 @@ def all_uids():
     return oauth.all_uids(app)
 
 
+@app.after_request
```

Comment:
Don't use a decorator. I don't like binding a dynamic variable to a static module value. During init just call: app.after/before_request(flask_before_request), which just registers the function.